packages/next/pages/index.tsx

```diff
@@ -0,0 +1,4 @@
+import 'setimmediate';
```

Comment:
I guess this is a polyfill? Can we add a comment for what it's being used for?

Reply:
I didn't run into any issues without it, but according to [this Expo guide](https://docs.expo.dev/guides/using-nextjs/#polyfill-setimmediate), it will most likely cause issues with libraries in the future if not polyfilled.